recipes/casatools/meta.yaml

```diff
@@ -0,0 +1,106 @@
+{% set version = "6.7.2.12" %}
+{% set build = 0 %}
+{% set mpi = "openmpi" %}  # force this ... for now?
+{% set mpi_prefix = "mpi_" + mpi %}
+{% set build_pin = mpi_prefix + '_*' %}
+
+package:
+  name: casatools
+  version: {{ version }}
+
+source:
+  fn: casa-{{ version }}.tar.gz
+  url: https://open-bitbucket.nrao.edu/rest/api/latest/projects/CASA/repos/casa6/archive?at=refs%2Ftags%2F{{ version }}&format=tgz
+  sha256: 056511bfe4b662e8f4972a7ba3232eb71c8ecedcee2a7db47ceeda2ec1fd8c0a
+  patches:
+    - macos-not-gcc.patch  # [osx]
+
+build:
+  number: {{ build }}
+  skip: true  # [win]
+  string: {{ mpi_prefix }}_py{{ py }}h{{ PKG_HASH }}_{{ build }}
+
+requirements:
+  build:
+    - cmake <4
+    - libgomp      # [linux]
+    - llvm-openmp 20  # [osx] ; match casacpp
```

Comment:
Please fix casacpp. I'm requesting the addition to host everywhere to get rid of these errorneous pins.

Reply:
Got it.